webpack.config.js

```diff
@@ -224,6 +231,7 @@ const config = {
         process.env.MB_EDITION === "ee"
           ? ENTERPRISE_SRC_PATH + "/overrides"
           : SRC_PATH + "/lib/noop",
+      embedding: EMBEDDING_SRC_PATH,
```

Comment:
This is used by both the SDK's webpack config and ESLint's `import/resolver` rule.